src/main/java/seedu/address/logic/parser/AddJobCommandParser.java

```diff
@@ -55,7 +55,7 @@ public AddJobCommand parse(String args) throws ParseException {
         JobDescription description = ParserUtil.parseDescription(argMultimap.getValue(PREFIX_DESCRIPTION).get());
         Set<Tag> requirements = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_REQUIREMENTS));
 
-        Job job = new Job(name, company, salary, description, requirements);
+        Job job = new Job(name, company, salary, description, requirements, null);
```

Comment:
Maybe can use empty set instead of null

Reply:
I used null as this is just a placeholder code, the capability of loading the association has yet to implemented. 

I have changed the null to a empty HashSet and add a todo comment for now.